### PR TITLE
chore: document the Commit Widget

### PR DESCRIPTION
**Pull request by levalleux-ludo**

### CODE REVIEW

File: src/components/widgets/commit/Commit.tsx [hunk 1]
```diff
@@ -32,10 +32,8 @@ export function Commit() {
   if (!productUuid && sellerId) {
     return <p>Missing 'productUuid' query param</p>;
   }
-  const lookAndFeel = searchParams.get("lookAndFeel") as "regular" | "modal";
-  if (!lookAndFeel) {
-    return <p>Missing 'lookAndFeel' query param</p>;
-  }
+  const lookAndFeel =
```
Comment (levalleux-ludo) [inline src/components/widgets/commit/Commit.tsx:35]: simplify the usage, making the lookAndFeel attribute optional for the integrator

File: public/scripts/boson-widgets.js [hunk 2]
```diff
@@ -215,6 +252,26 @@ function bosonWidgetReload(onLoadIframe) {
 }
 bosonWidgetReload();
 
+function bosonWidgetShowCommit(args, onLoadIframe) {
+  const paramsObject = toCleanedObject([
+    { tag: "sellerId", value: args.sellerId },
+    { tag: "configId", value: args.configId },
+    { tag: "account", value: args.account },
+    { tag: "productUuid", value: args.productUuid },
```
Comment (albertfolch-redeemeum) [inline public/scripts/boson-widgets.js:260]:
```suggestion
    { tag: "productUuid", value: args.bundleUuid },
    { tag: "productUuid", value: args.productUuid },
```

Comment (albertfolch-redeemeum) [reply public/scripts/boson-widgets.js:260]: incoming phygitals

File: public/scripts/boson-widgets.js [hunk 3]
```diff
@@ -128,6 +135,36 @@ function bosonWidgetReload(onLoadIframe) {
...
+        showCommitId.attributes[constants.lookAndFeelTag]?.value;
+      const modalMargin =
+        showCommitId.attributes[constants.modalMarginTag]?.value;
+      const account = showCommitId.attributes[constants.accountTag]?.value;
+      bosonWidgetShowCommit(
+        {
+          sellerId,
+          configId,
+          account,
+          productUuid,
```
Comment (albertfolch-redeemeum) [inline public/scripts/boson-widgets.js:159]: same as below

**Review by albertfolch-redeemeum — APPROVED** [review]
we need to add bundleUuid everywhere we mention productUuid as phygitals are coming soon, although maybe not that soon, so fyi

File: public/scripts/boson-widgets.js (hunk 2 above)
Comment (levalleux-ludo) [reply public/scripts/boson-widgets.js:264]: Done

File: public/scripts/boson-widgets.js (hunk 3 above)
Comment (levalleux-ludo) [reply public/scripts/boson-widgets.js:162]: Done

**Pull request merged**